.github/workflows/tests.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Tests
+
+on:
+  push:
+    branches:
+    # Check all branches
+  pull_request:
+    # Check all PR
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: [3.7, 3.8]
```

Comment:
We should test all Python versions >=3.6.
At the moment, the required Python version for MDTools as defined in setup.py is >=3.6 and <4.0.

Reply:
Good point, for now I just started with two versions for testing so that we don't run a large amount of actions while playing around. We can easily add lower versions as needed later.
But I don't think that 3.9 makes sense. AFAIK MDAnanlysis 1.1.1 does not support 3.9 yet (see [MDA discussion here](https://github.com/MDAnalysis/mdanalysis/issues/2974) and [pip requirements](https://pypi.org/project/MDAnalysis/)).

Reply:
Ok, then we are limited to Python versions not higher than 3.8. Probably I should change this accordingly in setup.py

Reply:
3.6 should be supported by MDAnalysis as well. I'll add it later. So setup.py should indicate somthing like >=3.5 and <= 3.9

Reply:
Added 3.6 to Actions and modified setup.py